src/bindgen/cdecl.rs

```diff
@@ -56,6 +56,25 @@ impl CDecl {
         cdecl.build_type(t, false);
         cdecl
     }
+
+    fn from_func_type(t: &Type, length_array: &Option<String>) -> CDecl {
```

Comment:
The name of this function makes it seem like `t` is a function type of sort. Should probably be `from_function_arg` or such.

Reply:
`length_array` should be named `array_length`, I think. Otherwise it seems like an array of some sort of length value.